tests/components/eq3btsmart/test_config_flow.py

```diff
@@ -0,0 +1,132 @@
+"""Test the eq3btsmart config flow."""
+
+from unittest.mock import patch
+
+from homeassistant import config_entries
+from homeassistant.components.bluetooth import BluetoothServiceInfoBleak
+from homeassistant.components.eq3btsmart.const import DOMAIN
+from homeassistant.const import CONF_MAC
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+from homeassistant.helpers.device_registry import format_mac
+from homeassistant.util import slugify
+
+from .const import MAC
+
+from tests.common import MockConfigEntry
+
+
+async def test_user_flow(hass: HomeAssistant) -> None:
+    """Test we can handle a regular successflow setup flow."""
+
+    result = await hass.config_entries.flow.async_init(
+        DOMAIN, context={"source": config_entries.SOURCE_USER}
+    )
+
+    with patch(
+        "homeassistant.components.eq3btsmart.async_setup_entry",
+        return_value=True,
+    ) as mock_setup_entry:
+        result = await hass.config_entries.flow.async_configure(
+            result["flow_id"],
+            {CONF_MAC: MAC},
+        )
+        await hass.async_block_till_done()
+
+    assert result["type"] == FlowResultType.CREATE_ENTRY
+    assert result["title"] == slugify(MAC)
+    assert result["data"] == {}
```

Comment:
```suggestion
    assert result["data"] == {}
    assert result["context"]["unique_id"] == MAC    
```

It would be good to assert the unique id is set as well

This suggestion may need to be adjusted